### DISCUSSION TITLE
[Mantine Dates] Why is dayjs a peer dependency?

### DISCUSSION THREAD

**Discussion by devrnt** [mantinedev/mantine#3391] [Q&A]
I was wondering what the reason is why `dayjs` is a peer dependency and not just a regular dependency for the Mantine Dates?

User-land is free to use any date library that they prefer, but by making `dayjs` a peer dependency, user-land is already pointed in a date library direction (this shouldn't prevent you from installing another one). When opening a `package.json` for the first time it's  confusing to see multiple date libraries.  Why not just ship dayjs, since it's really small like [you said](https://github.com/mantinedev/mantine/discussions/198#discussioncomment-1149323)?

**Comment by rtivital** [comment]
So, why is react a peer dependency, can we ship it as well?

**Reply by devrnt** [reply]
Those are 2 different things, I think you know that.
 It was a genuine question. The date picker emits a web standard Date object, so why should I manually have `dayjs` installed? It's exclusively used for internal usage of the picker.

**Reply by rtivital** [reply]
> Those are 2 different things, I think you know that.

No, I do not understand that, please explain how are these libraries different.

> It's exclusively used for internal usage of the picker.

No, it is not, dayjs can be used for localization. Like described in documentation.
